resource/specs/dev.toml

```diff
@@ -53,7 +58,14 @@ files = [
 name = "secp256k1_blake160_multisig_all"
 files = [
   { bundled = "specs/cells/secp256k1_data" },
-  { bundled = "specs/cells/secp256k1_blake160_multisig_all" },
+  { version = "v0.5.4", bundled = "specs/cells/secp256k1_blake160_multisig_all" },
+]
+
+[[genesis.dep_groups]]
+name = "secp256k1_blake160_multisig_all_v2"
+files = [
+  { bundled = "specs/cells/secp256k1_data" },
+  { version = "v0.6.0", bundled = "specs/cells/secp256k1_blake160_multisig_all" },
```

Comment:
I need to add a new field: ckb-system-scripts's `version` to `DepGroup`.
`ckb-system-scripts v0.5.4`'s `specs/cells/secp256k1_blake160_multisig_all` is legacy multisig.
`ckb-system-scripts v0.6.0`'s `specs/cells/secp256k1_blake160_multisig_all` is v2 multisig.

@driftluo @zhangsoledad @quake @doitian